return-3-signs.py

```diff
@@ -0,0 +1,9 @@
+from pickletools import string1
```

Comment:
why this import?

Reply:
import is my mistake, there shouldn't be this